docs/charts/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.2.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.17.0"
+appVersion: "1.471.0"
```

Comment:
Does it need to be the same as Casdoor version? Is it good to hard code it? What if new release comes out?

Reply:
Helm [recommends](https://helm.sh/docs/topics/charts/#the-appversion-field) matching the application version with the same value. Regarding new releases, some automation would need to be put in place (e.g. Github Action). If you prefer otherwise, that's fine as well.
>  the appVersion field is ... a way of specifying the version of the application. For example, the drupal chart may have an appVersion: "8.2.1", indicating that the version of Drupal included in the chart (by default) is 8.2.1. This field is informational, and has no impact on chart version calculations